files/image_config/hostcfgd/hostcfgd

```diff
@@ -304,13 +304,51 @@ class HostConfigDaemon:
             add = False
 
         self.iptables.iptables_handler(key, data, add)
+        
+    def feature_status_handler(self, key, data):
+        status_data = self.config_db.get_table('FEATURE')
+        for key in status_data.keys():
+            if not key:
+                syslog.syslog(syslog.LOG_WARNING, "FEATURE key is missing")
+                return
+        status = status_data[key]['status']
+        if not status:
+            syslog.syslog(syslog.LOG_WARNING, "status is missing for {}".format(key))
+            return
+        if status == "enabled":
+            start_cmds=[]
+            start_cmds.append("sudo systemctl enable {}".format(key))
+            start_cmds.append("sudo systemctl start {}".format(key))
+            for cmd in start_cmds:
+                syslog.syslog(syslog.LOG_INFO, "Running cmd - {}".format(cmd))
+                try:
+                    subprocess.check_call(cmd, shell=True)
+                except subprocess.CalledProcessError as err:
```

Comment:
Please check when this expect happens, if the function returns

Reply:
It doesn't return by default. Added explicit return Good catch.